docs_user/assemblies/assembly_configuring-network-for-RHOSO-deployment.adoc

```diff
@@ -4,43 +4,24 @@ ifdef::context[:parent-context: {context}]
 
 :context: configuring-network
 
-= Configuring the network for the RHOSO deployment
+= Configuring the network for the {rhos_long_noacro} deployment
 
-With {rhocp_long}, the network is a very important aspect of the deployment, and
-it is important to plan it carefully. The general network requirements for the
-{rhos_prev_long} ({OpenStackShort}) services are not much different from the ones in a {OpenStackPreviousInstaller} deployment, but the way you handle them is.
-
-[NOTE]
-For more information about the network architecture and configuration, see
-link:https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/18.0-dev-preview/html/deploying_red_hat_openstack_platform_18.0_development_preview_3_on_red_hat_openshift_container_platform/assembly_preparing-rhocp-for-rhosp[_Deploying Red Hat OpenStack Platform 18.0 Development Preview 3 on Red Hat OpenShift Container Platform_] and link:https://docs.openshift.com/container-platform/4.15/networking/about-networking.html[About
-networking] in _OpenShift Container Platform 4.15 Documentation_. This document will address concerns specific to adoption.
-
-// TODO: update the openstack link with the final documentation
-
-// TODO: should we parametrize the version in the links somehow?
-
-When adopting a new {OpenStackShort} deployment, it is important to align the network
+When you adopt a new {rhos_long} deployment, you must align the network
 configuration with the adopted cluster to maintain connectivity for existing
 workloads.
 
-The following logical configuration steps will incorporate the existing network
-configuration:
+You need to incorporate the existing network configuration in the following steps:
```

Comment:
```suggestion
Perform the following tasks to incorporate the existing network configuration:
```
Suggestion to avoid using "steps" here cos it isn't a procedure, sounds more like a workflow.